packages/minter-contracts/ligo/src/fixed_price_sale/README.md

```diff
@@ -217,3 +226,19 @@ Respective allowlisted contract is [`fixed_price_sale_market_tez_allowlisted.mli
 * `Sell` entrypoint fails with `SALE_ADDRESS_NOT_ALLOWED` in case any `token_for_sale_address` is not from the allowed list.
 
 Other entrypoints are not modified.
+
+<a name="Fixed-fee-extension"></a>
+## Fixed Fee extension
+
+The `fixed_fee` contract versions allow for the configuration of a fixed perecent of any sale to be paid to a fixed address. 
+
+``` ocaml
+type fee_data = 
+  [@layout:comb]
+  {
+    fee_address : address;
+    fee_percent : nat;
+  }
+```
+
+`fee_percent` is a nat between 0 and 100 representing the percent of the sale to be paid to the `fee_address`. If `fee_percent > 100`, the `sell` and `buy` entrypoints will both fail with `FEE_TOO_HIGH`.  Fee is calculated as `((fee_percent * price) / 100)`. 
```

Comment:
Great that this is mentioned in the spec!

One thing I would also explicitly mention: there are two possible models of how fees are accounted for. Let's say fees form 30%, and the seller declared a price of 100 money; whether will `(fees, seller_received, buyer_payed)` be `(30, 100, 130)` (fees are added to the declared price) or `(30, 70, 100)` (fees are already included into the price) here?

I see the answers to this question in the code but would be really handy to have it mentioned in the specification too.